src/game/puppyprint.c

```diff
@@ -86,48 +79,36 @@ s32 ramsizeSegment[NUM_TLB_SEGMENTS + 1] = {
     0, 0, 0,
     0, 0, 0
 };
-s8  audioRamViewer = FALSE;
 s32 mempool;
+u32 gPoolMem;
+u32 gPPSegScroll = 0;
+u32 gMiscMem = 0;
+struct CallCounter gPuppyCallCounter;
 
 extern u8 _mainSegmentStart[];
 extern u8 _mainSegmentEnd[];
 extern u8 _engineSegmentStart[];
 extern u8 _engineSegmentEnd[];
 extern u8 _framebuffersSegmentBssStart[];
 extern u8 _framebuffersSegmentBssEnd[];
+extern u8 _zbufferSegmentBssStart[];
+extern u8 _zbufferSegmentBssEnd[];
 extern u8 _buffersSegmentBssStart[];
 extern u8 _buffersSegmentBssEnd[];
 extern u8 _goddardSegmentStart[];
 extern u8 _goddardSegmentEnd[];
```

Comment:
many of these segments are defined already in [segment_symbols.h](https://github.com/HackerN64/HackerSM64/blob/6117cc6813df52de695daa813c80bfe5b590bf99/include/segment_symbols.h), I think it would be better to use those macros for any other extra segments you need and include this file